src/SerilogTraceListener/SerilogTraceListener.cs

```diff
@@ -79,7 +81,7 @@ public override bool IsThreadSafe
 
         public override void Close()
         {
-            logger = new SilentLogger();
+            this.logger = Log.Logger.ForContext("SourceContext", context);
```

Comment:
I don't think this will do much, most of the time - perhaps instead we could just make `Close()` a no-op? This would avoid having to track `this.context` as well.